content/rendering-with-batch/monitor/optional_fis.md

```diff
@@ -0,0 +1,198 @@
+    - COUNT(n) - Run the action on the specified number of targets, chosen from the identified targets at random. For example, COUNT(1) selects one of the targets.
+  - **actions**: The actions for the experiment, in this case, sending a 2-minute Spot Interruption notice to the selected instances. For more information, see [Actions](https://docs.aws.amazon.com/fis/latest/userguide/actions.html) in the Fault Injection Simulator User Guide .
+  - **stopConditions**: Specifies a stop condition for an experiment template.  In this case, we don't have any.
+  - **roleArn**: The Amazon Resource Name (ARN) of an IAM role that grants the FIS service permission to perform service actions on your behalf.
+
+Execute this command to create the FIS template from the JSON defined above.
+
+```
+export FIS_TEMPLATE=$(aws fis create-experiment-template --cli-input-json file://fis-experiment.json | jq -r '.experimentTemplate.id')
+echo "FIS Template ID: ${FIS_TEMPLATE}"
+```
+
+{{% notice info %}}
+You should wait a few minutes after starting the job above for enough EC2 instances to have joined the pool before starting the FIS experiment.
+{{% /notice %}}
+
+At any time before, during, or after the FIS experiment, you can look at your AWS EC2 Spot instance request status with the following command
+
+```
+aws ec2 describe-spot-instance-requests --query 'SpotInstanceRequests[].[InstanceId, Status.Code]'
+```
+
+The output will show you the EC2 instance IDs and the Spot instance request state:
+  - **fulfilled**: indicates and EC2 spot request was fulfilled and is still active
+  - **instance-terminated-by-experiment**: indicates that the EC2 Spot instance received the FIS Interruption signal and was terminated
+  - **instance-terminated-by-user**: indicates that the EC2 Spot instance fulfilled it's job and was terminated by AWS Batch after no longer being needed.
+
+
+
+Make sure that you have at least 3 EC2 Spot Requests that are **fulfilled** using the command from above, before you execute this command to start the FIS experiment from the template.  If desired, you can run this command at several points during the AWS Batch run to simulate multiple Spot Interruptions.
+
+```
+export FIS_EXPERIMENT=$(aws fis start-experiment --experiment-template-id ${FIS_TEMPLATE} | jq -r '.experiment.id')
+echo "FIS Experiment ID: ${FIS_EXPERIMENT}"
+```
+
+{{% notice note %}}
+When started the FIS experiment starts an asynchronous process to send the interruption signal to 5 EC2 instances
+{{% /notice %}}
+
+You can check the status of the FIS Experiment's progress with the following command
+
+```
+aws fis get-experiment --id ${FIS_EXPERIMENT}  | jq -r '.experiment.state'
+```
+
+If the status is:
+- **running**: the job is still underway
+- **completed**: the FIS experiment has been run and instances should respond to the interruption notice.
+- **failed**: the FIS Experiment has failed and the reason will help you understand why
```

Comment:
line 127 makes a reference to a reason, where can that reason be found?
Also, let's be consistent about dots at the end of a line